internal/cmd/tui.go

```diff
@@ -196,6 +197,10 @@ func tuiCmd() *cobra.Command {
 				return err
 			}
 
+			if len(blocks) == 0 {
+				return errors.New("no scripts in README.md")
```

Comment:
As a file might have a different name, let's do this:

```suggestion
				return errors.Errorf("no scripts in %s", fFileName)
```